src/main/webapp/dev/js/common/instructorFeedbackResults.es6

```diff
@@ -240,6 +240,19 @@ function getAppendedResponseRateData(data) {
     return appendedResponseStatus;
 }
 
+function showHideSectionDisplayMode() {
+    const selectedView = $('#viewSelect').val();
+    const selectedSection = $('#sectionSelect').val();
+
+    if (selectedView && selectedSection && selectedView === 'question' && selectedSection !== 'All') {
```

Comment:
To make it flow more logically, let's make it `selectedView && selectedView === 'question' && ...`.